gradle.properties

```diff
@@ -15,9 +15,7 @@ gradleNexusPublishPluginVersion=1.3.0
 grailsAsyncVersion=6.0.0-M1
 grailsVersion=7.0.0-SNAPSHOT
 groovyVersion=4.0.24
-hibernateValidatorVersion=7.0.5.Final
```

Comment:
I think this is still required because data mapping references it

Reply:
org.hibernate.validator:hibernate-validator version comes from grails-bom -> spring-boot-dependencies.